### PR TITLE
fix: tokenRenewal seems to be hard coded #1540

### PR DESCRIPTION
**Pull request by regevbr**
fix: #1540

### CODE REVIEW

**Comment by regevbr** [comment]
@NGPixel can you please CR? Super short fix...

**Comment by regevbr** [comment]
@NGPixel can you please CR? Super short fix...

File: server/core/auth.js [hunk 1]
```diff
@@ -112,7 +113,8 @@ module.exports = {
       if (err) { return next() }
 
       // Expired but still valid within N days, just renew
-      if (info instanceof Error && info.name === 'TokenExpiredError' && moment().subtract(14, 'days').isBefore(info.expiredAt)) {
+      if (info instanceof Error && info.name === 'TokenExpiredError' &&
+        moment.subtract(ms(WIKI.config.auth.tokenRenewal), 'ms').isBefore(info.expiredAt)) {
```
Comment (NGPixel) [inline server/core/auth.js:117]: Should be `moment()`, otherwise you're relying on an existing instance which could have been modified previously.

Comment (regevbr) [reply server/core/auth.js:117]: Ops my bad

Comment (regevbr) [reply server/core/auth.js]: @NGPixel fixed, please review

**Pull request merged**